queries.yaml

```diff
@@ -168,3 +168,7 @@ post_process:
     source_layer: landuse
     label_property_name: label_position
     label_property_value: "yes"
+  TileStache.Goodies.VecTiles.transform.generate_label_features:
```

Comment:
Same deal @zerebubuth points out in #273 that we're not currently setup to run for more than 1 layer.